src/ds/d400/d400-color.cpp

```diff
@@ -64,16 +64,16 @@ namespace librealsense
         // except for D405, in which the color is part of the depth unit
         // and it will then been found in end point 0 (the depth's one)
         auto color_devs_info_mi3 = filter_by_mi(group.uvc_devices, 3);
-        if (color_devs_info_mi3.size() == 1 || ds::RS457_PID == _pid)
+        if (color_devs_info_mi3.size() == 1 || _is_mipi_device)
         {
             // means color end point in part of a separate color sensor (e.g. D435)
-            if (ds::RS457_PID == _pid)
+            if (_is_mipi_device)
```

Comment:
Do we have it set already? needed for D430?
I suggest checking this code with D430 GMSL too as we don't have it on CI and we may break it